CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### 📝 Documentation
 
+* Add release runbook on RELEASING.md ([#2533](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2533))
```

Comment:
```suggestion
* Document the instructions for a release ([#2533](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2533))
```

Reply:
The goal is not simply to record the steps, but to create a standard process that people can easily follow. Changed the message to 'Document release instructions'